db/seeds.rb

```diff
@@ -34,8 +34,8 @@
 
 users = user_content.map { |content| User.create(content) }
 
-
-users.first.orders.create(status:0, cart: {"1"=>3, "4"=>1, "9"=>2})
+user = users.first
+user.orders.create(status:0, cart: {"1"=>3, "4"=>1, "9"=>2})
```

Comment:
Space missing after colon.<br>Surrounding space missing for operator '=>'.<br>Space inside { missing.<br>Space inside } missing.